model/event/event_finder.go

```diff
@@ -39,6 +39,19 @@ func Find(coll string, query db.Q) ([]EventLogEntry, error) {
 	return events, nil
 }
 
+func FindPaginated(coll string, query db.Q, page, limit int) ([]*EventLogEntry, error) {
```

Comment:
The last 2 params should just be combined into just "skip" because you aren't actually returning a specific page from this function

Reply:
I think we are missing `query.Limit(limit)` to return a single page of results where the page size is limit

Reply:
Please ignore my previous comment. I realize that limit is being handled in `MostRecentHostEvents`.